<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="LessonBlindScriptT" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="author" content="Jack D. Thatcher, Ph.D., West Virginia School of Osteopathic Medicine" />
    <title>Lesson Script</title>
</head>

<body>
    <div aria-hidden="true">
        <p>This script is provided for use by the visually impaired.  It is intended to be read by a text to speech translator.  Many of the words and abbreviations are spelled phonically for correct pronunciation by the translator.</p>
    </div>
    <div aria-hidden="false">
        <p><h3><b>Linkage Analysis</b></h3><br>
                If one homolog has dominant alleles for two loci, and the other homolog has recessive alleles, then the genes are in the cis conformation.<br>
                If one homolog has a dominant allele for one locus and a recessive for the other, while the other homolog has recessive and dominant alleles, then the genes are in the trans conformation.<br>
                Crossing over switches the conformation.<br>
                Recombination is any time separate DNA strands are ligated together.  Crossing over is a natural form of recombination.<br>

            <b>Linkage</b><br>
                To understand linkage let's compare an example where the loci are far apart to an example where they are close together.<br>
                If the loci are far apart then numerous crossovers are likely to occur between them.<br>
                If the loci are right beside each other then there is very little chance that a crossover will happen to occur between them.<br>
                Now consider an intermediate distance.<br>
                In this case there is enough distance for a crossover to occur between the loci, but the chance is not as great as when the loci are far apart.<br>
                Thus, the chance of recombination between loci is proportional to the distance between them.<br>

            <b>Linkage Analysis</b><br>
                To learn how to analyze linkage data consider a cross between two loci.  One parent is homozygous recessive for both loci, and a parent who is heterozygous for both.  Basically, this is a double test cross.<br>
                There will be four classes of progeny.<br>  
                The first two classes are cis, dominant phenotype for both loci, recessive for both.<br>
                The other two class are trans, dominant for the first locus, recessive for the second, and recessive for the first and dominant for the second.<br>
                The first step to analyzing linkage data is to identify the parental type progeny and recombinant progeny.<br>
                The parental type progeny are those who were produced from gametes who did not undergo a crossover between the loci.  These will always be the classes with the highest numbers.<br>
                The recombinant progeny are those who were produced from a crossover between the loci.  The number of progeny in these classes will be lower.<br>
                This is because the distance between the loci is so short that the chance that a crossover occurred between them is low.<br>
                Thus, if the parental type classes are those who are dominant for both loci and recessive for each, then the heterozygous parent was in the cis conformation.<br>
                If the parental type classes are those who are dominant for one loci and recessive for the other, then the heterozygous parent was in the trans conformation.<br>

            <b>Map Unit Calculation</b><br>
                Map units equal 100 x ( sum of Crossovers Between Loci divided by the Total ) <br>
                This is the formula for calculating the distance between loci.<br>  
                Note that it is a calculation of the percent recombination between the loci.<br>
                One map unit is defined as 1% recombination.<br>
                The working version of the formula is Map units equal 100 x ( sum of Recombinant Type F1 divided Total ) <br>
                For example, consider a cross where there were 48 and 50 F1 progeny in the two parental type classes, and 13 and 11 in the recombinant classes<br>
                The total recombinant progeny would be 13 plus 11 equal to 24.<br>
                The parental type progeny would be 48 plus 50 equal to 98.<br>
                The total would be 24 plus 98 equal to 122.<br>
                The recombination fraction would be 24 divided by 122 equal to 0.197.<br>
                The percent recombination would be 0.197 x 100 equal to 19.7%.<br>
                Thus, the loci would be 19.7 map units apart.<br>
                What's are the maximum number of map units that can be calculated between two loci? <br>
                To answer this question let's consider an example.<br>
                One crossover between two loci switches the conformation.<br>
                A second crossover returns the loci to their original conformation.<br>
                A third crossover returns the loci to the switched conformation.<br>
                Thus, an odd number of crossovers switches the conformation, an even number maintains the original conformation.<br>
                Thus, the maximum distance between two loci that can be calculated directly is 50 map units.  This is because if they are 50 map units or more apart there is an equal chance of having an odd or even number of crossovers between them, and the recombination will always be 50%.<br>
                We're now ready to define linkage.  If two loci are less than 50 map units apart they are linked.<br>
                If two loci are more than 50 map units apart they cannot be mapped directly because there can never be more than 50% recombination between them.<br>
                Linked Genes are less than 50 map units apart.<br>
                If they were not linked then the numbers for all four F1 classes would be approximately the same.<br>
                Note linkage is a violation of independent assortment, which predicts 50% recombination.<br>  

            <b>Sample Problem, Three Point Cross</b><br>
                Now let's see what can be deduced from a three point cross, where three loci are tracked.<br>
                As an example, weâ€™ll perform a cross with three mutant genes.  These mutations will be curly (cu), whose mutants have upward curved wings, hairy (h) with bristly thoraces, heads and wings, and rough (row) with disrupted ommatidial arrays.<br>
                The cross will be written + cu + h + row with cu cu  h h  row row<br>
                The F1 are scored and the following numbers are observed.<br>
                36 curly flies<br>
                33 hairy rough<br>
                31 curly hairy<br>
                28 rough<br>
                17 hairy<br>
                12 curly rough<br>
                7 wild type<br>
                3 curly hairy rough<br>
                This data was sorted from the most numerous classes to the least.  This facilitates distinguishing parental type progeny from recombinants and double crossovers.<br>
                The two most abundant classes are cu + +, and + h row, so these are the parental types.<br>
                Notice how one parental class demonstrates the curly phenotype, while the other is hairy and rough.<br> 
                This tells us the conformation of the heterozygous parent.<br>
                hairy & rough where on one parental chromatid (trans to each other), while curly was on the other (cis to hairy & rough).<br>
                The next four counts constitute two recombinant classes.<br>
                What about the last two; what produced such low numbers? <br>
                The smallest groups resulted from double crossovers.<br>
                The chance of two crossovers occurring, one between hairy and curly, and another between curly and rough, is less than the chance for one crossover.<br>  
                The odds actually equal the chance of one crossover times the chance of the other.<br>
                We're now ready to deduce another property, the order of these loci.<br>
                Notice how in the heterozygous parent curly was cis to hairy & rough, but in the double crossover group it wound up trans.<br>
                Thus, the only way two crossovers could move curly to the same homolog as hairy & rough is if it is between hairy & rough.<br>
                Therefore, order of the loci must be hairy, curly, rough, with curly between hairy and rough.<br>
                In the first recombinant class a crossover occurred between the hairy & curly loci.<br>  
                In the second recombinant class a crossover occurred between curly & rough.<br>
                In the double crossover class a crossover occurred between hairy & curly, and another occurred between curly & rough.<br>  
                We're now ready to compute the distance between the loci. Remember, a map unit equals percent recombination.<br>  
                Remember the formula, map units equal 100 times the sum of all Crossovers Between Loci divided by the Total.<br>
                Understand, the double crossovers must be incorporated into the calculation for each recombinant class.  This is because each double crossover represents a crossover in each region.<br>  
                For instance, a double crossover involved a crossover between hairy & curly, so it has to be added to the calculation of the hairy curly distance.  However, the second crossover occurred between curly & rough, so it must also be added to the curly rough calculation.<br>
                The calculations for the distance between hairy and curly is are + 28 + 7 + 3 divided by 36 + 33 + 31 + 28 + 17 + 12 + 7 + 3, equal to 69 divided by 167, equals 41 map units.<br>
                The calculations for the distance between curly and rough are 17 + 12 + 7 + 3 divided by 167 equal to 39 divided by 167  equals 23 map units.<br>
                There is one other parameter that can be deduced from this data.  This is the coefficient of interference.<br>  
                To understand this parameter, you have to realize that when a crossover occurs between two linked loci, it decreases the chances of a second crossover occurring within an adjacent region.  The coefficient of interference, abbreviated C I, is an estimate of extent of this phenomenon.  It is determined by calculating the ratio between observed and expected double crossovers.<br>
                The observed double crossover is determined by simply calculating the fraction of double crossovers.<br>
                The fraction of observed double crossers equals 7 + 3 divided by 167, which equals 10 divided by 167, which is 0.06. <br>
                The expected is calculated by multiplying the chance of a crossover within one region times the chance of a crossover in the second region.<br>
                The chance of crossovers occurring between the two region have already been calculated.<br>
                It is 0.41 for the region between hairy and curly, 0.23 for curly and rough.<br>
                Thus, the expected fraction of double crossovers is 0.41 x 0.23 =  0.096.<br>
                Dividing observed by expected gives a parameter known as the coefficient of coincidence, which is used to determine the coefficient of interference.<br>
                The Coefficient of Coincidence is 0.06 divided by 0.096, which equals 0.62.<br>
                The coefficient of interference is one minus the coefficient of coincidence.<br>
                The Coefficient of Interference is 1 minus 0.62, which equals 0.38.<br>
                To summarize, the order of the loci is hairy, curly, rough.<br>
                In the heterozygous parent curly was cis to hairy and rough.  Hair and rough were trans to each other.<br>
                The distance between hair and curly is 41 map units.<br>
                The distance between curly and rough is 23 map units.<br>
                A crossover between either region inhibits a second crossover within the adjacent region by 38%.<br>
                Note that the distance between hairy and rough is 43 plus 23 equal to 63 map units.<br>
                This is greater than 50 map units so they could not have been mapped by linkage analysis directly.  However, by determining the distance between each and an intermediate locus, and adding these together, the total distance can be deduced.<br>

            <b>Linkage Maps</b><br>
                Linkage maps, also known as genetic maps, show the relative positions of multiple genes in a genome.<br>
                Not only do they record the map units between the genes, they demonstrate the chromosome the genes are located on.<br>  
                Maps like these provide approximations of the location of genes on chromosomes, but they are not 100% accurate.  This is because the frequency of crossing over varies along the chromosome.<br>
                It is not practical or ethical to perform linkage crosses with humans so other methods, such as lod scores, have been employed to estimate the position of genes.  However, due to genomic sequencing the exact position of most genes has been determined, but it is still important to understand the property of linkage.  For instance, it is utilized for genetic markers.<br>
        </p>  
    </div> 
</body>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
<script src="../../../../../BioME%20Sara/CommonFiles/masterUIcontrol.js"></script>
</html>
